config.yml

```diff
@@ -64,3 +64,6 @@ notebooks:
     packages_pyodide:
       - mat3ra-esse
       - mat3ra-made
+  - name: specific_examples
```

Comment:
Let’s put a comment that this will be applied for the whole folder

Reply:
no, you specify this line in the installation function